README.md

```diff
@@ -503,34 +221,25 @@ Accepted log types:
 	1. To test CLI changes locally you can run `node <project_dir>/src/bin/medic-conf.js`. This will run as if you installed via npm.
 	1. To test changes that are imported in code run `npm install <project_dir>` to use the local version of medic-conf.
 
-## compress images
-
-To compress PNGs and SVGs in the current directory and its subdirectories, two commands are available:
-
-	compress-pngs
-	compress-svgs
-
-## Wishlist
-
-* only upload things which have changed (this could be a separate mode - e.g. `update` vs `configure`)
-
 # Releasing
 
-## Do I need to update `release-notes.md`?
-
-As we strive to have clear, readable commit messages for every change, [release-notes.md](./release-notes.md) should not be updated for every single change. Instead, it should be used as a forum for deep information about significant changes, breaking changes, changes to interfaces, changes in behavior, new feature details, etc.
-
-## Performing the release
-
 1. Create a pull request with prep for the new release. This should contain changes to release notes if required and anything else that needs to be done:
 1. Get it reviewed and approved
 1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate. This will:
-  - Update versions in `package.json` and `package-lock.json`
-  - Commit those changes locally and tag that commit with the new version
-  - "Compile" and publish the changes to npm
+    - Update versions in `package.json` and `package-lock.json`
+    - Commit those changes locally and tag that commit with the new version
+    - "Compile" and publish the changes to npm
 1. `git push && git push --tags` to push the npm generated commit and tag up to your pre-approved pull request
 1. Merge the pull request back into master
 
+Note that as we strive to have clear, readable commit messages for every change, [release-notes.md](./release-notes.md) should not be updated for every single change. Instead, it should be used as a forum for deep information about significant changes, breaking changes, changes to interfaces, changes in behavior, new feature details, etc.
+
+# Build Status
+
+Builds brought to you courtesy of [Travis CI](https://travis-ci.org/medic/cht-conf).
+
+<a href="https://travis-ci.org/medic/medic-conf"><img src="https://travis-ci.org/medic/medic-conf.svg?branch=master"/></a>
+
 # Copyright
 
 Copyright 2013-2018 Medic Mobile, Inc. <hello@medicmobile.org>
```

Comment:
Not going to 2019 this while you're here? Or 2020 for forward compatibility ;-)

Reply:
Good call, but I'll stick to making it current.

```suggestion
Copyright 2013-2019 Medic Mobile, Inc. <hello@medicmobile.org>
```